gemviz/bluesky_runs_catalog.py

```diff
@@ -17,7 +17,7 @@
 
 from . import utils
 
-PAGE_START = -1
+PAGE_START = 0
 PAGE_SIZE = 10
```

Comment:
Looking at this with fresh eyes, wonder if controlling the number if items displayed to the number of visible whole rows might be a better choice of page size?  Then vertical scroll bar would not ever be needed.

<img width="1053" height="611" alt="Image" src="https://github.com/user-attachments/assets/907d4580-6d50-4511-a778-da2fa2dbd90a" />

A thought for a future issue.

Reply:
added

- #232